Library/ViewModels/PledgeViewModelTests.swift

```diff
@@ -151,9 +151,9 @@ final class PledgeViewModelTests: TestCase {
     self.vm.outputs.goToApplePayPaymentAuthorization.map { $0.merchantIdentifier }
       .observe(self.goToApplePayPaymentAuthorizationMerchantId.observer)
 
-    self.vm.outputs.goToThanks.map(first).observe(self.goToThanksProject.observer)
```

Comment:
Ah I remember you messaged me about these. Were `first`, `second` and `third` all having issues compiling?